src/main/resources/collaboratorPage.json

```
@@ -0,0 +1,154 @@
+{
+  "page": {
+    "title": "Team",
+    "subtitle": "Collaborators",
+    "description": "Our collaborators are the backbone of our community, offering vital expertise and support that drive our initiatives forward and enhance our digital presence."
+  },
+  "contact": {
+    "title": "Contact us",
+    "links": [
+      {
+        "type": "email",
+        "link": "london@wcc.com"
+      },
+      {
+        "type": "slack",
+        "link": "http://shortlink_to_slack.com"
+      }
+    ]
+  },
+  "collaborators": [
+    {
+      "fullName": "Lily N.",
+      "position": "UX/UI Designer & Engineer",
+      "images": [
+        {
+          "path": "https://womencodingcommunity.com/assets/images/collaborators/1.jpeg",
+          "alt": "Collaborator profile image",
+          "type": "desktop"
+        }
+      ],
+      "network": [
+        {
+          "type": "linkedin",
+          "link": "https://www.linkedin.com/in/lilynguyendk"
+        },
+        {
+          "type": "github",
+          "link": "https://github.com/lilyndk"
+        }
+      ],
+      "memberType": "MEMBER"
```

```suggestion
      "memberType": "COLLABORATOR"
```

Also please update the enum like this: 
``` 
public enum MemberType {
    DIRECTOR,
    LEADER,
    EVANGELIST,
    VOLUNTEER,
    MENTOR,
    MENTEE,
    MEMBER,
    SPEAKER,
    COLLABORATOR,
}
```